README.md

```diff
@@ -135,7 +135,7 @@ I would like to thank to my coding partners.
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍 Just like I have made my logo which you can see in the image above ⬆️